code/_onclick/item_attack.dm

```diff
@@ -66,6 +78,19 @@
 		return TRUE
 	return FALSE //return TRUE to avoid calling attackby after this proc does stuff
 
+/**
+ * Called on the item before it hits something, when right clicking.
+ *
+ * Arguments:
+ * * atom/target - The atom about to be hit
+ * * mob/living/user - The mob doing the htting
+ * * params - click params such as alt/shift etc
+ *
+ * See: [/obj/item/proc/melee_attack_chain]
+ */
+/obj/item/proc/pre_attack_alt(atom/target, mob/living/user, params)
```

Comment:
could we call this "secondary" or something of the likes that won't be confused with the alt key?

Reply:
This proc is following the convention that was put forth when the first ones were created, I think changing the names of all of them is out of scope for this PR.

Reply:
I'll open a PR to change the other names